laevatein/src/main/java/com/laevatein/internal/utils/PhotoMetadataUtils.java

```diff
@@ -78,21 +79,21 @@ public static Point getBitmapSize(ContentResolver resolver, Uri uri, Activity ac
         return new Point((int) (w * widthScale), (int) (h * heightScale));
     }
 
-    public static Point getBitmapBound(ContentResolver resolver, Uri uri) {
+    private static Point getBitmapBound(ContentResolver resolver, Uri uri) {
+        BitmapFactory.Options options = new BitmapFactory.Options();
+        options.inJustDecodeBounds = true;
         InputStream is = null;
         try {
-            BitmapFactory.Options options = new BitmapFactory.Options();
-            options.inJustDecodeBounds = true;
-            is = resolver.openInputStream(uri);
-            BitmapFactory.decodeStream(is, null, options);
-            int width = options.outWidth;
-            int height = options.outHeight;
-            return new Point(width, height);
+            is = ContentResolverWrapper.openInputStream(resolver, uri);
         } catch (FileNotFoundException e) {
             return new Point(0, 0);
         } finally {
             CloseableUtils.close(is);
         }
+        BitmapFactory.decodeStream(is, null, options);
```

Comment:
I want comment here like "`BitmapFactory.decodeStream` 's argument "InputStream" is nullable, so never crash here".

Reply:
👍

Reply:
I added a comment here. thank you.